OutOfSchool/OutOfSchool.Common/Validators/CustomAgeAttribute.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace OutOfSchool.Common.Validators;
+
+/// <summary>
+/// Validation attribute for dates. It allows only dates that are at least <see cref="MinAge"/> and at most <see cref="MaxAge"/>.
+/// This attribute can be used on <see cref="DateTime"/> or <see cref="DateOnly"/> types.
+/// </summary>
+[AttributeUsage(AttributeTargets.Property | AttributeTargets.Field | AttributeTargets.Parameter, AllowMultiple = false)]
+public class CustomAgeAttribute : DataTypeAttribute
+{
+    public CustomAgeAttribute()
+        : base(DataType.Date)
+    {
+    }
+
+    /// <summary>
+    /// Gets the minimum age of the persom in years.
```

Comment:
(Mistype) `persom` is expected to have "n" at the end - perso**n**. 

Same in `MaxAge` property description.